src/main/java/it/pagopa/interop/probing/eservice/operations/model/Eservice.java

```diff
@@ -104,7 +104,8 @@ public class Eservice implements Serializable {
   @NotNull
   @Enumerated(EnumType.STRING)
   @Column(name = "state")
-  private EserviceState state;
+  private EserviceStateBE state;
```

Comment:
If it is not null maybe it need a default?

Reply:
no, I would expect it to return an error in this case. otherwise, what value to assign? it would be inconsistent data